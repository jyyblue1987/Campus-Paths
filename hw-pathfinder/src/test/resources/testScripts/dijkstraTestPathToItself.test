# Find a path to itself with Dijkstra's algorithm

CreateGraph graph

AddNode graph A
AddNode graph B

AddEdge graph A B 1.0

FindPath graph A A