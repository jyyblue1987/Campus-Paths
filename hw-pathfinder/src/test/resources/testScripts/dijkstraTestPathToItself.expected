# Find a path to itself with Dijkstra's algorithm

created graph graph

added node A to graph
added node B to graph

added edge 1.000 from A to B in graph

path from A to A:
